AWS/legos/aws_get_all_service_names/aws_get_all_service_names3.json

```diff
@@ -1,11 +1,11 @@
 {
```

Comment:
We should keep the directory name and .py & .json file names the same.
AWS/legos/aws_get_all_service_names/aws_get_all_service_names3.json ---> AWS/legos/aws_get_all_service_names/aws_get_all_service_names.json

Also, the action_entry_function is the same as the directory name.